### PR TITLE
Advanced autobiography

### PR DESCRIPTION
**Pull request by SoultexLesley**
Autobiogrphy improved by some divisions and different style settings.

### CODE REVIEW

File: Homework_advanced_autobiography2/style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ {	 
+  text-align:center;
+  border: 1px none;
+  background: rgba(10 , 0 , 0 , 0.8);
+  width:85%;
+  margin:0px auto;  
+  padding-bottom:30px;
+  border-radius:0 0 40px 40px;
+  padding-top:10px;
+
```
Comment (aserheichuk-amplify) [inline Homework_advanced_autobiography2/style.css]: remove useless blank line

File: Homework_advanced_autobiography2/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<html>
+ <head>
+  <title>Резюме</title>
+  <meta charset="utf-8"/>
+  <meta name="author"  content="Olkhovyk Boris"/>
+  <link rel="stylesheet" href="style.css"/>
+ </head>
+ <body>
+  <div class="header_div" >
+  <h1>Резюме</h1>
```
Comment (aserheichuk-amplify) [inline Homework_advanced_autobiography2/index.html:10]: this tag is only one on page, and this is block tag, why do you wrap it into `div`?

**Comment by SoultexLesley** [comment]
Maybe now it's better than it was?

**Comment by SoultexLesley** [comment]
:+1:

**Pull request merged**